src/controllers/users.controller.ts

```diff
@@ -60,6 +60,18 @@ class UsersController {
       next(error);
     }
   };
+
+  public manageGroups = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Type should be added.
`public manageGroups = async (req: Request, res: Response, next: NextFunction): Promise<void>`